### ISSUE TITLE
bower packageManager

### ISSUE THREAD

**Issue by Toilal** [tjunnone/npm-check-updates#145]
Thanks for bower support.

There's still reference to npm in messages when using bower packageManager option.

```
$ ncu -m bower
...
The following dependencies are satisfied by their declared version range, but the installed versions are 
behind. You can install the latest versions without modifying your package file by using npm update. If 
you want to update the dependencies in your package file anyway, use ncu --upgradeAll.
...
```

**Comment by metaraine** [comment]
Thank you. I got rid of most of the other references.

Does `bower update` work the same as `npm update`? If so, I can just change the text.

**Comment by Toilal** [comment]
I think it's the same: http://bower.io/docs/api/#update

**Comment by metaraine** [comment]
Fixed in v2.3.2.
